org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/compiler/ImportManager.java

```diff
@@ -66,8 +67,8 @@ public ImportManager(boolean organizeImports, JvmDeclaredType thisType, char inn
 		this.organizeImports = organizeImports;
 		this.innerTypeSeparator = innerTypeSeparator;
 		if (thisType != null) {
-			thisTypeSimpleNames.add(thisType.getSimpleName());
-			thisTypeQualifiedNames.add(thisType.getQualifiedName(innerTypeSeparator));
+			getThisTypeSimpleNames().add(thisType.getSimpleName());
```

Comment:
You need to use the getters because you want to override them?

Reply:
I need to have access to the fields' values in my subtype of ImportManager in order to code a specific addImport function for a Python generator dedicated to my DSL.

Reply:
I'm inclined to go with local field access and protected, final getters to make the field-values available to subtypes.